ldi-core/ldes-client-v2/request-executor/src/main/java/ldes/client/requestexecutor/executor/clientcredentials/ClientCredentialsRequestExecutor.java

```diff
@@ -25,9 +30,11 @@ public Response execute(Request request) {
 
 		try (com.github.scribejava.core.model.Response response = oAuthService.execute(oAuthRequest)) {
```

Comment:
I might have missed the bigger context.. but how come we have two Response objects?

Reply:
The oauth lib has a Response object as well.. Since the scope of that object is limited to this class only, I did not pick a different name for our Response object.